bindata/v4.1.0/config/aws-overrides.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kubecontrolplane.config.openshift.io/v1
+kind: KubeAPIServerConfig
```

Comment:
What happens when we don't need the override in the future?  Operator responsible for deleting this file?

Reply:
Nothing, this doesn't create a new file. This is only an override for the already existing config map we use to pass configuration to a kas instance. 

See https://github.com/p0lyn0mial/cluster-kube-apiserver-operator/blob/31509c40ae05f548377ef02a2697a84594920624/pkg/operator/targetconfigcontroller/targetconfigcontroller.go#L217

Reply:
it's a sparse merge-patch like overlay